internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicManagerStats.java

```diff
@@ -81,5 +88,9 @@ final void registerTopicMetadataFetcherSensors(TopicMetadataFetcher topicMetadat
         new AsyncGauge(
             (ignored, ignored2) -> topicMetadataFetcher.getConsumerWaitListSize(),
             "consumer_wait_list_size"));
+    registerSensorIfAbsent(
+        new AsyncGauge(
+            (ignored, ignored2) -> pubSubAdminOpFailureCount.getAndSet(0),
+            "pub_sub_admin_op_failure_count"));
```

Comment:
Do we have any central registry for the metrics we have? If so, can you document about this new metric?

Reply:
I don't think we have a central repo. But I'll update the commit message.